moneo.py

```diff
@@ -30,6 +31,7 @@ def deploy(self):
             ('true' if self.args.profiler_metrics else 'false') + '"'
 
         print('Deployment type: ' + self.args.type)
+        logging.info('Moneo starting, Deployment type: ' + args.type)
```

Comment:
How about creating two `handler` (for record log into console and file) instead of using print and logging?

Reply:
Thats fair. I will revisit this in a later PR.